### ISSUE TITLE
'auto_io.h' not found

### ISSUE THREAD

**Issue by aornugent** [JuliaPCL/PCLIO.jl#1]
Hi, I've been trying to build JuliaPCL for Ubuntu and am having I'm having some trouble loading PCLIO:

```
In file included from :1:
__cxxjl_13.cpp:4:10: fatal error: 'pcl/io/auto_io.h' file not found
#include <pcl/io/auto_io.h>
         ^
LLVM ERROR: Program used external function '_ZN5boost3argILi8EEC1Ev' which could not be resolved!
```

Can you suggest where 'auto_io.h' should come from?
Thanks,
A

**Comment by r9y9** [comment]
Thank you for the report! And sorry, the packages are actually not ready to use for end users yet. For now I was unable to build PCL packages on linux (see https://github.com/JuliaPCL/PCL.jl/issues/16 for details). 

As for the 'auto_io.h', it should come from system-installed PCL headers (e.g. `/usr/local/include/pcl-1.8/pcl/io/auto_io.h`) if you already have installed PCL. If not, `Pkg.build("LibPCL")` should installed the headers in `~/.julia/v0.5/LibPCL/deps/usr/include/...`.

**Comment by r9y9** [comment]
I wrote a installation guide: https://juliapcl.github.io/latest/installation.html. I haven't try PCL on linux yet, but I hope it helps for users.

**Comment by ghost** [comment]
Woah, that was speedy! I'll try again with PCL 1.8 and let you know how I get on, thanks very much for your help.

**Comment by aornugent** [comment]
Problem resolved with PCL 1.8.
